modules/ROOT/pages/iam-limits.adoc

```diff
@@ -0,0 +1,24 @@
+= Limits in Access Management
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Access Management imposes the following limits on various features in an organization:
+
+* Roles per client: 3000
+* Clients per root organization: 10000
```

Comment:
This seems maybe incorrect to me.

```
ms_authentication> select org_id, count(*) as num_clients from clients group by org_id having count(*) >= 10000 order by count(*) desc;
+--------------------------------------+---------------+
| org_id                               | num_clients   |
|--------------------------------------+---------------|
| ee85d185-ed55-419a-8ca0-fda90872691d | 14182         |
| 99838324-71ae-4681-94f3-5425c6ed5ffd | 10001         |
| 315666df-0540-4afc-a4c3-158c5275ab91 | 10000         |
| b2473331-fff9-45aa-b3f4-f8dfba5d20df | 10000         |
+--------------------------------------+---------------+

ms_authentication> select sub_organization_ids from organizations where id = '7f34e134-ef86-4a88-a34a-368f44ff5920';
-[ RECORD 1 ]-------------------------
sub_organization_ids | {3e646885-33af-42ab-987f-b602c33841f7,129d1685-60c9-4eab-b506-5042c06012c9,b486fec7-3340-4b7e-b5fa-27fee86def4c,d0da16a1-974f-4fb0-baa4-9149c2e5a586}

ms_authentication> select created_at from clients where org_id in ('3e646885-33af-42ab-987f-b602c33841f7'::uuid,'129d1685-60c9-4eab-b506-5042
 c06012c9'::uuid,'b486fec7-3340-4b7e-b5fa-27fee86def4c'::uuid,'d0da16a1-974f-4fb0-baa4-9149c2e5a586'::uuid) order by created_at desc limit 10
 ;
+----------------------------+
| created_at                 |
|----------------------------|
| 2021-11-10 16:12:07.211681 |
| 2021-11-10 06:39:40.796051 |
| 2021-11-09 23:54:26.233054 |
| 2021-11-08 16:10:27.194359 |
| 2021-11-05 11:11:52.938565 |
| 2021-11-05 10:39:09.826806 |
| 2021-11-05 09:41:00.744449 |
| 2021-11-05 09:31:04.193474 |
| 2021-11-01 11:23:52.974624 |
| 2021-11-01 09:08:59.927887 |
+----------------------------+
```

So you can see the customer is able to still create clients on the BGs, even though the root org is at its limit.

Reply:
can this be changed to `Clients per root organization or sub organization`